### ISSUE TITLE
Extract particular values from the query

### ISSUE THREAD

**Issue by bagder** [curl/trurl#77]
May like this?

~~~
$ trurl "https://example.com?search=hello&utm_source=tracker&more=data" -g '{query:search}'
hello
~~~

**Comment by bagder** [comment]
Based on #88 and #90 also existing now, we should create a generic function to do the conversion from string to an array of `name=value` to operate on as we keep adding ideas for working on individual pairs. And of course a second function that brings the array back into a string for when we have to put it back into the URL.

**Comment by bagder** [comment]
Fixed via #103
